packages/components/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import {
+  DisclosureProps,
+  Menu as AriakitMenu,
+  MenuButton,
+  MenuItem,
+  useMenuState,
+} from "ariakit";
+import map from "lodash/map";
+import { Button } from "../Button";
+import { Box } from "../../primitives/Box";
+
+export type MenuItemProps = {
+  label: string;
+  onClick: () => void;
+};
+
+export type MenuProps = {
+  /** The button element to open the menu and display the list of menu items. */
+  menuButton?: JSX.Element;
+  /** The list of menu items. */
+  items: MenuItemProps[];
+};
+
+const EllipsisButton = (
```

Comment:
I wonder if we could call this "VerticalEllipsisButton"

In mobile, web, and general application design, the vertical ellipsis, ⋮, is sometimes used as an interface element, where it is sometimes called a [kebab icon](https://en.wikipedia.org/wiki/Kebab_icon). [source](https://en.wikipedia.org/wiki/Ellipsis#:~:text=In%20mobile%2C%20web%2C%20and%20general,sometimes%20called%20a%20kebab%20icon.)

Reply:
Or "KebabButton" :)